### ISSUE TITLE
Order roles alphabetically on the admin

### ISSUE THREAD

**Issue by Piedone** [OrchardCMS/OrchardCore#16795]
### Is your feature request related to a problem? Please describe.
/Admin/Roles/Index will show a seemingly random order of roles:

![{FBC208C3-14EE-469C-8C5A-D7E8DC591800}](https://github.com/user-attachments/assets/6a4630e0-55b0-4caf-b1e8-fa0861fa7dc5)

This makes it harder to find the role you want to manage.

### Describe the solution you'd like
Order the roles alphabetically.

### Describe alternatives you've considered
I can't think of any.

**Comment by gvkries** [comment]
Agreed, it's even worse, because saving a role always changes the order and that's super confusing.

**Comment by matTrabolsi** [comment]
Hello, I’ve submitted a pull request to fix the issue. You can find the PR [here](https://github.com/OrchardCMS/OrchardCore/pull/16796), 
This is my first contribution to an open-source project, and I’m also new to .NET.
